src/discordtipbot/discordbot.cr

```diff
@@ -401,10 +407,44 @@ class DiscordBot
     @tip.get_config(guild_id(msg), "mention") || false
   end
 
+  def lucky(msg)
+    return reply(msg, "**ERROR**: This command doesn't work in DMs") if private?(msg)
+
+    cmd_usage = "#{@config.prefix}lucky [amount]"
+
+    # cmd[0]" command, cmd[1]: amount"
+    cmd = msg.content.split(" ")
+
+    return reply(msg, cmd_usage) unless cmd.size > 1
+
+    amount = amount(msg, cmd[1])
+    return reply(msg, "**ERROR**: You have to specify an amount! #{cmd_usage}") unless amount
+
+    return reply(msg, "**ERROR**: You have to lucky rain at least #{@config.min_tip} #{@config.coinname_short}") unless amount >= @config.min_tip
+
+    trigger_typing(msg)
+
+    users = active_users(msg).to_a
+
+    return reply(msg, "**ERROR**: There is no one to make lucky!")
+
+    user = users.sample
+
+    case @tip.transfer(from: msg.author.id, to: user, amount: amount, memo: "lucky")
+    when "success"
+      return reply(msg, "#{msg.author.username} luckily rained **#{amount} #{@config.coinname_short}** onto **<@#{user}>**")
```

Comment:
Each of these `return` is not needed.

Reply:
btw this isn't the only place I've done this... You missed it in `#soak` and `#rain` and `#tip`